db/re/item_combo_db.txt

```diff
@@ -76,6 +76,7 @@
 1620:2509:19139,{ .@r = getequiprefinerycnt(EQI_HAND_R); bonus bMatkRate,1; if (.@r >= 5) { bonus bMatkRate,2; if (.@r >= 7) { bonus bMatkRate,2; } } }
 1620:2535,{ bonus bMatkRate,5; bonus2 bSubEle,Ele_Neutral,25; }
 1620:19020,{ .@r = getequiprefinerycnt(EQI_HAND_R); bonus bMatk,.@r*10; if (.@r >= 10) { autobonus "{ bonus bVariableCastrate,-50; }",100,5000,BF_MAGIC; /* Confirm: Success rate? */ } }
+1625:18542,{ .@r = getequiprefinerycnt(EQI_HAND_R); bonus bHealPower,3*(.@r/2); }
```

Comment:
```suggestion
1625:18542,{  bonus bHealPower,getequiprefinerycnt(EQI_HAND_R); }
```
(you re-wrote the combo with holy stick (5920))

Reply:
Thanks